wiki/Guides/Discord_Rich_Presence/zh.md

```diff
@@ -0,0 +1,38 @@
+# Discord Rich Presence
+
+自 2017-10-31 起，Discord Rich Presence 已经加入稳定版！
+
+## Rich Presence 意味着什么？
+
+这意味着您现在可以在 Discord 服务器内显示并加入osu!多人游戏，查看好友在osu!中做什么，甚至可以请求加入目前正在进行的游戏当中！您还可以观看服务器内的其他玩家。
+
+## Rich Presence 如何工作？
+
+- **首先打开您的游戏客户端更新游戏！** 这项全新的 Discord 功能仅包含在2017年万圣节特别版本以及之后更新的版本中 - 如果您还在使用旧版本那么它将**不会工作**。
+- 确保 Discord 在您的系统上将osu!检测为已注册游戏 - 通常情况下这项操作会被默认执行，因此您无须担心。您可以在 Discord 设置菜单中的 `App设置` -> `游戏动态` 中进行检查。
+- 在 `游戏动态` 选项中打开 `将正在运行的游戏显示为个人状态`。您可以不加入别人正在显示的游戏，但您需要显示自己的游戏。
+- 加入任意多人游戏房间中，打开您的 Discord 客户端并在聊天框附近找到这个图标 ![the shiny green button in front of the plus sign](img/shiny-green-button.jpg)。 如果您刚开始玩游戏，可能需要等待几秒钟的时候才会显示。
```

Comment:
alt text should be translated ("the shiny green button in front of the plus sign")